assets/base.css

```diff
@@ -3353,6 +3353,11 @@ details-disclosure > details {
     transform: rotate(0.5deg); /* Less intense rotation for collage items. */
   }
 
+  .animate--hover-3d-lift .grid__item:hover,
+  .animate--hover-3d-lift .collage__item:hover {
+    z-index: 1; /* Make sure the hovered card is the topmost card. */
+  }
```

Comment:
For general shadows we're applying some padding to account for the shadow spread. Which we're not with this and it can cut off the shadow, specifically on sliders: [video](https://screenshot.click/28-21-amct2-zh7o9.mp4)

Something similar could be useful to look into. Maybe as a follow up for the next release.

Reply:
Yeah, this would be a good followup. There's a bug with complementary products where the shadow doesn't extend out of its container: 

<img width="239" alt="Screenshot 2023-04-28 at 4 13 05 PM" src="https://user-images.githubusercontent.com/1202812/235245640-d5cdd0d2-b4cd-484d-967d-9c07c6f203cb.png">